scripts/error_codes.py

```diff
@@ -213,32 +239,34 @@ def main(argv):
         ["libevmasm", "liblangutil", "libsolc", "libsolidity", "libsolutil", "libyul", "solc"],
         [".h", ".cpp"]
     )
-    used_ids = get_used_ids(source_file_names)
+    source_id_to_file_names = find_ids_in_source_files(source_file_names)
 
     ok = True
-    for id in sorted(used_ids):
+    for id in sorted(source_id_to_file_names):
         if len(id) != 4:
             print(f"ID {id} length != 4")
             ok = False
         if id[0] == "0":
             print(f"ID {id} starts with zero")
             ok = False
-        if used_ids[id] > 1:
-            print(f"ID {id} appears {used_ids[id]} times")
+        if len(source_id_to_file_names[id]) > 1:
+            print(f"ID {id} appears {len(source_id_to_file_names[id])} times")
```

Comment:
```suggestion
            print(f"ID {id} appears {len(source_id_to_file_names[id])} times.")
```

Reply:
Do you mean the full stop? There are some other messages without it.

Do we have any policy? My rule of thumb would be to avoid the full stop for single sentences, but use for multi-sentence texts.